### DISCUSSION TITLE
Maybe should change the `Math` class name

### DISCUSSION THREAD

**Discussion by YaoKaiLun** [dolanmiu/docx#2535] [General]
`Math` is a standard built-in object, Maybe should change the `Math` component class name
https://github.com/dolanmiu/docx/blob/master/src/file/paragraph/math/math.ts

**Comment by dolanmiu** [comment]
So is `Document`

Unfortunately, there will be clashes. We can't account for all of them

Correctly namespace, and it will all be fine
